components/camel-debezium/src/main/docs/debezium-component.adoc

```diff
@@ -0,0 +1,217 @@
+[[debezium-component]]
+= Debezium Component
+
+*Available as of Camel version 3.0*
+
+The Debezium component is wrapper around https://debezium.io/[Debezium] using https://debezium.io/docs/embedded/[Debezium Embedded], which enabled Change Data Capture from various databases without the need of Kafka or Kafka Connect.
+
+*Note on handling failures:* Per https://debezium.io/docs/embedded/#handling_failures[Debezium Embedded Engine] documentation, the engines is actively recording source offsets and periodically flushes these offsets to a persistent storage, so when the application is restarted or crashed, the engine will resume from the last recorded offset.
+Thus, at normal operation, your downstream routes will receive exactly once event, however in case of application crash (not having a graceful shutdown), the application will resume from the last recorded offset,
```

Comment:
```suggestion
Thus, at normal operation, your downstream routes will receive each event exactly once, however in case of an application crash (not having a graceful shutdown), the application will resume from the last recorded offset,
```